app/MMB/src/components/sectionListScrollTo/OnLayout.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import type { OnLayout as OnLayoutType } from '@kiwicom/mobile-shared';
+
+type State = {| height: number |};
+type Props = {| +children: State => React.Node |};
+
+export default class OnLayout extends React.Component<Props, State> {
```

Comment:
We already have this component, it is Exported as Dimensions from the shared package

Reply:
Dimensions is taking the dimension of the whole window, as opposed to the dimensions of the element it seems.
<img width="267" alt="screen shot 2018-07-31 at 15 01 31" src="https://user-images.githubusercontent.com/22741774/43461032-aee41600-94d2-11e8-9dcc-b33c49417862.png">

Reply:
ahh, sorry, I misunderstood the usecase

Reply:
Then could you please rename the children prop to render in this case?

Reply:
Of course!